packages/@lwc/wds-core/src/node/plugins/lwc/compile.js

```diff
@@ -16,7 +16,7 @@ export const VIRTUAL_TEMPLATE_EMPTY = '/virtual/empty.html';
 const VIRTUAL_CONTENT = new Map(
   Object.entries({
     [VIRTUAL_CSS_EMPTY]: {
-      body: '',
+      body: 'export default null;',
```

Comment:
If a `componentName.css` file does not exist alongside `componentName.js`, the virtual/empty CSS file is what is provided. This wasn't working before, which is why an empty body was not a problem. Now that the virtual/empty CSS file is being imported, it needs to provide a falsey default export in order for the ES module system to allow it to be imported elsewhere.